jerry-core/lit/lit-strings.h

```diff
@@ -25,15 +25,81 @@
 #define LIT_BYTE_NULL (0)
 
 /**
+ * For the formal definition of Unicode transformation formats (UTF) see Section 3.9, Unicode Encoding Forms in The
+ * Unicode Standard (http://www.unicode.org/versions/Unicode7.0.0/ch03.pdf#G7404, tables 3-6, 3-7).
```

Comment:
"ECMAScript source text is represented as a sequence of characters in the Unicode character encoding, version 3.0 or later."
Definitions in this file are the same for both Unicode 3.0 and 7.0.
We can change this link to 3.0.